uniffi_aries_vcx/core/Cargo.toml

```diff
@@ -17,9 +17,12 @@ path = "uniffi-bindgen.rs"
 
 [dependencies]
 uniffi = { version = "0.23.0", features = ["cli"] }
-aries-vcx = { path = "../../aries_vcx" }
-test_utils = {path = "../../tools/test_utils" }
-tokio = { version = "1.24.1" }
+aries-vcx = { path = "../../aries_vcx", features = [
+    "vdrtools_wallet",
+    "credx",
+] }
+test_utils = { path = "../../tools/test_utils" }
```

Comment:
Just a heads-up that the `test-utils` was added as a full dependency to provide the `MockLedger` type. It's really meant to only exist as a `dev-dependency`, although this crate might not need it at all unless it's used for testing.

Reply:
@swaptr can it be removed?

Reply:
Yes  we can safely remove it now.